.docs/PROJECT-STRUCTURE.md

```diff
@@ -1,57 +1,57 @@
-# Project File Stucture
+# Project File Structure
 
-This ASCII tree represents the Project Stucture. Read through the comments to know about the Project Stucture.
+This ASCII tree represents the project structure. Read through the comments to learn more about the walking-pal-web repo.
 
 ```sh
 .
-├── .docs #contains all markdown files and assets used to write the documentation files.
-├── assets/ #contains assets used in the WebApp
+├── .docs #markdown files and assets used to write the documentation files
+├── assets/ #assets used throughout walking-pal-web
 │   ├── gif/
-│   │   ├── some-gif.gif #All gif, png, svg names must be in kebab-case
-│   │   ├── index.ts #This imports all the gif files in the `gif/` directory and exports them.
+│   │   ├── some-gif.gif #gif, png, svg names must be in kebab-case
+│   │   ├── index.ts #imports all the gif files in the `gif/` directory and exports them
 │   │   └── ...(other gifs)
 │   ├── png
 │   └── svg
-├── components/ #contains global components (used by more than one pages)
+├── components/ #global components (used by more than one page)
 │   ├── GlobalComponent1/
-│   │   ├── GlobalComponent1.tsx #Component files are in PascalCase
-│   │   ├── global-component1.module.scss #SCSS files are in kebab-case
-│   │   ├── index.ts #this file re-exports the React Component from GlobalComponent1.tsx (along with any other accompanied exports)
-│   │   └── componentUtility.ts #this might have any utility function used by the GlobalComponent1 React Component
+│   │   ├── GlobalComponent1.tsx #component file names are in PascalCase
+│   │   ├── global-component1.module.scss #SCSS file names are in kebab-case
+│   │   ├── index.ts #re-exports the React component from GlobalComponent1.tsx (along with any other accompanied exports)
+│   │   └── componentUtility.ts #utility function used by the GlobalComponent1 React component
 │   └── GlobalComponent2/
-├── hooks/ #contains global custom hooks (used by more than one pages)
+├── hooks/ #global custom hooks (used by more than one page)
 │   └── useWindowResize.ts
 ├── pages/
-│   ├── api/ #contains our Nextjs api endpoints
-│   │   ├── route1/ #this is route1 endpoint
+│   ├── api/ #Nextjs API endpoints
+│   │   ├── route1/ #route1 endpoint
 │   │   │   └── index.pg.ts #route1 endpoint logic
-│   │   ├── utils/ #miscellaneous utilites used by endpoints
-│   │   └── index.pg.ts #contains root (/) endpoint handler
+│   │   ├── utils/ #miscellaneous utilities used by endpoints
+│   │   └── index.pg.ts #root (/) endpoint handler
 │   ├── page-name1/
-│   │   ├── components/ #contains local components (used by PageName1 page only)
```

Comment:
`PageName1` refers to the components. `page-name1` is the page-route/folder-name. So intead you can write `(used by PageName1 page component only`

Reply:
I changed the PageName1 section back to normal